tests/hazmat/backends/test_openssl.py

```diff
@@ -456,6 +458,32 @@ def test_unsupported_oaep_label_decrypt(self):
                 )
             )
 
+    def test_invalid_oaep_decryption(self):
```

Comment:
Arguably this test exists to trigger a specific msg on the error stack in the openssl backend, but it definitely should raise the proper `ValueError` no matter the backend. We can move it (although I'd keep an altered version of the comment so we understand its value in the future) if you'd like.